README.md

```diff
@@ -101,25 +101,25 @@ This will try to pre-shuffle the data so that we only need to call the VE in one
 more performant due to avoiding a coalesce/shuffle afterwards.
 
 ```
---conf spark.com.nec.spark.preshuffle-partitions=8
+--conf spark.cyclone.spark.preshuffle-partitions=8
 ```
 
 ## Configuration options
 
 Note, _default specified in the `=`_
 
 ```
---conf spark.com.nec.spark.aggregate-on-ve=true
---conf spark.com.nec.spark.sort-on-ve=false
---conf spark.com.nec.spark.project-on-ve=true
---conf spark.com.nec.spark.filter-on-ve=true
---conf spark.com.nec.spark.exchange-on-ve=true
---conf spark.com.nec.spark.pass-through-project=false
---conf spark.com.nec.spark.fail-fast=false
---conf spark.com.nec.spark.join-on-ve=false
---conf spark.com.nec.spark.amplify-batches=true
---conf spark.com.nec.spark.ve.columnBatchSize=<spark col batch size>
---conf spark.com.nec.spark.ve.targetBatchSizeMb=64
+--conf spark.cyclone.spark.aggregate-on-ve=true
```

Comment:
Possible to rename these options to spark.cyclone.sql.xxx-on-ve?

Reply:
Sure

Reply:
Done